Automattic/simplenote-android
# Add Tablet Dimensions

### Fix
Add dimensions for the **_Style_** screen so that the style list items are full-width on phones and have a maximum width on tablets.  See the screenshots below for illustration.

Phone|Tablet
-|-
<kbd><a href="https://user-images.githubusercontent.com/3827611/99316838-45348100-2822-11eb-9e6a-bd2a4b66678f.png"><img src="https://user-images.githubusercontent.com/3827611/99316838-45348100-2822-11eb-9e6a-bd2a4b66678f.png"></a></kbd>|<kbd><a href="https://user-images.githubusercontent.com/3827611/99316864-4ebde900-2822-11eb-982f-facfc27ec1c3.png"><img src="https://user-images.githubusercontent.com/3827611/99316864-4ebde900-2822-11eb-982f-facfc27ec1c3.png"></a></kbd>
<kbd><a href="https://user-images.githubusercontent.com/3827611/99316859-4cf42580-2822-11eb-8b65-a2a642e1e593.png"><img src="https://user-images.githubusercontent.com/3827611/99316859-4cf42580-2822-11eb-8b65-a2a642e1e593.png"></a></kbd>|<kbd><a href="https://user-images.githubusercontent.com/3827611/99316871-5087ac80-2822-11eb-8800-4c3f84231711.png"><img src="https://user-images.githubusercontent.com/3827611/99316871-5087ac80-2822-11eb-8800-4c3f84231711.png"></a></kbd>

Since a `RelativeLayout` view was added as the top-level view to all the `style_list_row_*.xml` layouts, the diff of these changes looks huge unless you use the "Hide whitespace changes" setting in GitHub.  That makes the changes go from (+370/-311) to (+71/-11).

### Test
When performing the test steps below, be sure to use a small screen device (i.e. phone) as well as large screen device (i.e. tablet) in both portrait and landscape orientation.
1. Open navigation drawer.
2. Tap **_Settings_** item in drawer.
3. Tap **_Style_** setting under **_Appearance_** section.
4. Notice **_Style_** screen as shown in screenshots above.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

#### Note
Contact me for an installable build.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.
